test/libyul/YulOptimizerTest.cpp

```diff
@@ -118,6 +118,7 @@ TestCase::TestResult YulOptimizerTest::run(ostream& _stream, string const& _line
 
 	updateContext();
 
+	auto& m_ast = m_object->code;
```

Comment:
Could also rename every instance of `*m_ast` to `*m_object->code`, it is about 30 occurances.